src/bootstrap/doc.rs

```diff
@@ -851,7 +851,7 @@ tool_doc!(
     in_tree = true
 );
 tool_doc!(Clippy, "clippy", "src/tools/clippy", ["clippy_utils"], in_tree = true);
-tool_doc!(Miri, "miri", "src/tools/miri", ["miri"], in_tree = false);
+tool_doc!(Miri, "miri", "src/tools/miri", ["miri"], in_tree = true);
```

Comment:
looks like these are all in-tree now :) so we can remove the option and hard-code `true` in the macro

Reply:
I will do these cleanups in a follow up, getting the PR merged quickly is important as you cannot rebase a subtree addition